### PR TITLE
Corp Num Field For DBA/Xpro Ltds

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6274

*Description of changes:*

* Removed Corp num field for BC DBA's.

* Removed Corp num field for XPRO Ltds when NOT completing on own behalf.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/common/applicant-info-3.vue [hunk 1]
```diff
@@ -126,7 +126,7 @@
 
       <v-row>
         <v-col cols="2" />
-        <v-col cols="5" v-if="showCorpNum">
+        <v-col cols="5" v-if="getShowCorpNum === 'colin'">
```
Comment (severinbeauvais) [inline src/components/common/applicant-info-3.vue:129]: Would you mind adding a const or enum for this value?

See also getter.ts lines 680+ and other code references to this.

(Thanks!)

Comment (cameron-freshworks) [reply src/components/common/applicant-info-3.vue:129]: Surreee

Comment (cameron-freshworks) [reply src/components/common/applicant-info-3.vue]: Updated in next commit! 😁

Comment (severinbeauvais) [reply src/components/common/applicant-info-3.vue]: Awesome! Another small step for mankind.

**Review by severinbeauvais — APPROVED** [review]

**Comment by andrepestana-aot** [comment]
Removing Corp num field for XPRO Ltds will not affect International Jurisdiction, right?

**Comment by cameron-freshworks** [comment]
> Removing Corp num field for XPRO Ltds will not affect International Jurisdiction, right?

In deployed environments the Corp Num field isn't required for XPRO International already , no?
It doesn't affect it, because it's only removing it from the Xpro Canada, if thats what you mean?

**Review by andrepestana-aot — APPROVED** [review]
Look good to me!

**Comment by andrepestana-aot** [comment]
> > Removing Corp num field for XPRO Ltds will not affect International Jurisdiction, right?
> 
> In deployed environments the Corp Num field isn't required for XPRO International already , no?
> It doesn't affect it, because it's only removing it from the Xpro Canada, if thats what you mean?

Yeah, I see... OK. Just confirming!

**Comment by cameron-freshworks** [comment]
> > > Removing Corp num field for XPRO Ltds will not affect International Jurisdiction, right?
> > 
> > 
> > In deployed environments the Corp Num field isn't required for XPRO International already , no?
> > It doesn't affect it, because it's only removing it from the Xpro Canada, if thats what you mean?
> 
> Yeah, I see... OK. Just confirming!

Thank you for asking though, i had to confirm as well :)

**Pull request merged**